src/js/components/Pagination/Pagination.js

```diff
@@ -15,25 +20,28 @@ const Pagination = forwardRef(
       numMiddlePages = 1, // number of pages surrounding the active page
       onChange,
       page: pageProp,
-      show = 1,
+      // ideating on prop to show a message like "Showing x-y of z items"
+      // showSummary,
+      show,
```

Comment:
Wondering if we need to bring consistency to the meaning of `show`? I believe we have mixed meanings currently. Here show represents the page to show, but in DataTable etc. it references the item index to show.

Reply:
That's a good point. I guess the way I was thinking of it, `show` was interpreted in the context of the component, meaning in DataTable you're asking to "show a piece of data" vs pagination is "show a page" -- however I think it's a good point especially in the context of Pagination + children. We can run with the direction we have currently, but note it as a discussion point for upcoming review.